src/views/candidate-quiz-listing.phtml

```diff
@@ -49,17 +49,29 @@ require_once('partials/head.phtml');
 
     <!--Pagination block-->
     <nav aria-label="Page navigation example">
-        <ul class="pagination justify-content-center">
-            <li class="page-item disabled">
-                <a class="page-link" href="#" tabindex="-1" aria-disabled="true">Previous</a>
-            </li>
-            <li class="page-item"><a class="page-link" href="#">1</a></li>
-            <li class="page-item active"><a class="page-link" href="#">2</a></li>
-            <li class="page-item"><a class="page-link" href="#">3</a></li>
-            <li class="page-item">
-                <a class="page-link" href="#">Next</a>
-            </li>
-        </ul>
+        <?php if ($pages['quizzesNumber'] > 5): ?>
```

Comment:
Consider moving the logic that decides the number of pages out of the html. Maybe send a $pagesCount too.